src/components/AssetsImagePreviewFilter/displayMessages.jsx

```diff
@@ -0,0 +1,11 @@
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+  assetsImagePreviewFilterLabel: {
+    id: 'assetsImagePreviewFilterLabel',
+    defaultMessage: 'Hide File Preview',
```

Comment:
TODO: Check in with Sylvia.

Reply:
Sylvia [said](https://openedx.atlassian.net/browse/EDUCATOR-2434?focusedCommentId=311150&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-311150) this is good to go.